emqx/cluster.hocon

```diff
@@ -0,0 +1,306 @@
+authentication = [
+  {
+    backend = http
+    body {
+      clientId = "${clientid}"
+      password = "${password}"
+      username = "${username}"
+    }
+    connect_timeout = "15s"
+    enable = true
+    enable_pipelining = 1
+    headers {
+      content-type = "application/json"
+    }
+    mechanism = password_based
+    method = post
+    pool_size = 1
+    request_timeout = "10s"
+    ssl {
+      ciphers = []
+      depth = 10
+      enable = false
+      hibernate_after = "5s"
+      log_level = notice
+      reuse_sessions = true
+      secure_renegotiate = true
+      verify = verify_peer
+      versions = [
+        "tlsv1.3",
+        "tlsv1.2"
+      ]
+    }
+    url = "http://host.docker.internal:3000/api/comms/v2/auth"
```

Comment:
Need to double check, but this hostname (host.docker.internal) was a mac only thing (or at leats part of Docker Desktop, the thing that technically you need a paid license for...) last time I checked

might need to be `172.17.0.1` which is the normal default